lite-pm/pages/components/HomePage.js

```diff
@@ -0,0 +1,143 @@
+
+handleChange(event) {
+  const target = event.target;
+  const value = target.type === "number" ? target.number : target.value;
+  const name = target.name;
+
+  this.setState({
+    [name]: target.value,
+  });
+}
+
+  toggleAddProjectModal() {
+    this.setState({ showAddProject: !this.state.showAddProject });
+  }
+  
+  addProject() {
+    this.toggleAddProjectModal();
+    this.createProject();
+    this.setState({
+      projectName: "Project Name",
+      projectDescription: "Description",
+      projectDuration: "Duration"
+    });
+  }
+
+  renderCreateProjectModal() {
+    return (
+      <Modal
+        isOpen={this.state.showAddProject}
+        toggle={this.toggleAddProjectModal}
+      >
+        <ModalHeader>Create a new project!</ModalHeader>
+        <ModalBody className="text-center">
+          <label>
+            <input
+              className={styles.inputs}
+              name="projectName"
+              type="text"
+              placeholder="Project Name"
+              onChange={this.handleChange}
+            />
+          </label>
+          <br />
+          <label>
+            <input
+              className={styles.inputs}
+              name="projectDescription"
+              type="text"
+              placeholder="Description"
+              onChange={this.handleChange}
+            />
+          </label>
+          <br />
+          <label>
+            <input
+              className={styles.duration}
+              name="projectDuration"
+              type="number"
+              placeholder="0"
+              onChange={this.handleChange}
+            />
+          </label>
+          <br />
+        </ModalBody>
+        <ModalFooter>
+          <Button color="secondary" onClick={this.toggleAddProjectModal}>
+            cancel
+          </Button>
+          <Button color="success" onClick={this.addProject}>
+            Create Project
+          </Button>
+        </ModalFooter>
+      </Modal>
+    );
+  }  
+
+  render() {
+    return (
+      <div>
+        <Navigationbar></Navigationbar>
+        <Container className={styles.container}>
+          <div>
+            <h1 className={styles.h1}>
+              The eastiest way to manage your small project
+            </h1>
+            <div className={styles.description}>
+              LitePM allows you to create and schedule tasks in an easy-to-use
+              Gantt chart, manage team members, and easily create and share
+              Google Docs with your team. All in a single page!
+            </div>
+          </div>
+          <div className={styles.homeImage}>
+            <img src="" alt="img"></img>
+          </div>
+          <Button color="secondary" onClick={this.toggleAddProjectModal}>
+            Create Project
+          </Button>
+        </Container>
+
+        <div>
+          <this.renderCreateProjectModal/>
+        </div>
+      </div>
+    );
+  }
+
+}
```

Comment:
Just want to put it out there that this modal was created by myself just happened to be over VS Code Live Share so Jason got the commit.